File: WebSites/Abastecimientos/App_Code/RequisicionItem.vb
Imports Microsoft.VisualBasic

Public Class RequisicionItem

    Public Property Id As Integer
        Get
            Return _id
        End Get
        Set(value As Integer)
            _id = value
        End Set
    End Property

    Public Property Codigo As String
        Get
            Return _codigo
        End Get
        Set(value As String)
            _codigo = value
        End Set
    End Property

    Public Property Nombre As String
        Get
            Return _nombre
        End Get
        Set(value As String)
            _nombre = value
        End Set
    End Property

    Public Property Cantidad As Decimal
        Get
            Return _cantidad
        End Get
        Set(value As Decimal)
            _cantidad = value
        End Set
    End Property


    Private _codigo As String
    Private _cantidad As Decimal
    Private _nombre As String
    Private _id As Integer
End Class

File: Projects/SINAB_Entidades/Tipos/RequisicionItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SINAB_Entidades.Tipos
{
    public class RequisicionItem
    {
        public int Id { get; set; }
        public string Codigo { get; set; }
        public string Nombre { get; set; }
        public decimal Cantidad { get; set; }
    }
}
